examples/streaming/wordcount/src/main/scala/org/apache/gearpump/streaming/examples/wordcount/WordCount.scala

```diff
@@ -64,18 +64,13 @@ object WordCount extends AkkaApp with ArgumentsParser {
     val debugMode = config.getBoolean("debug")
     val sleepSeconds = config.getInt("sleep")
 
-    val localCluster = if (debugMode) {
-      val cluster = new EmbeddedCluster(akkaConf: Config)
-      cluster.start()
-      Some(cluster)
+    val runtimeEnv = if (debugMode) {
```

Comment:
Is the "debugMode" option necessary ?  If examples are run in IDE, they are already in debug mode.